redesign/case-studies/inclusive-classrooms/skills-wheel.html

```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Find out more about the data collection and visualistion app that we built for Inclusive Classroom to help them help young people">
```

Comment:
Find out more about the data collection and visualisation education app that we built for Inclusive Classrooms